cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -102,20 +131,49 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left {
+            .u-webfont-medium();
+            display: inline-block;
+            float: none;
+            font-size: 1.125em;
+            grid-area: category;
+            line-height: 1.33333;
```

Comment:
This value looks like the result of a calculated value. Was this copied from production code by chance?

Reply:
No, it represents a `line-height` of 24px for a `font-size` of 18px. Is there a more preferable way to express it?

Reply:
Prolly do the calculation `line-height: 24px / 18px`. I defer to @Scotchester for variables that would help here.

Reply:
You'd have to do `unit(24px / 18px)` to drop the unit.

Reply:
But it'd probably be better to condense the `font-size`, `font-weight`, and `font-height` properties here into a call of [the `.heading-4()` mixin](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L120-L129).